src/client.ts

```diff
@@ -180,9 +180,13 @@ export interface ReactSDKClient extends Omit<optimizely.Client, 'createUserConte
   getForcedDecision(decisionContext: optimizely.OptimizelyDecisionContext): optimizely.OptimizelyForcedDecision | null;
 
   fetchQualifiedSegments(options?: optimizely.OptimizelySegmentOption[]): Promise<boolean>;
+
+  getQualifedSegments(): string[] | null;
```

Comment:
This must be an API for UserContext. Do we need this at client level?

Reply:
I've updated